### PR TITLE
Show a shuttle's current route on its popup

### PR DESCRIPTION
**Pull request by JCBird1012**
Pretty self-explanatory. Also fixed a quick typo in `model.go`.

Also I'm not quite sure why commits 206ac67 and 3a8ae81 are including themselves in this PR... They've already been merged to master? However, the files changed still represents the relevant changes.

Let me know if there's a different way you'd like to present this information to the user.

### CODE REVIEW

File: api/vehicles.go [hunk 1]
```diff
@@ -184,9 +185,13 @@ func (App *API) UpdateMessageHandler(w http.ResponseWriter, r *http.Request) {
 				log.WithError(err).Error("Could not load time zone information.")
 				continue
 			}
-			lastUpdate := update.Created.In(loc).Format("3:04:05pm")
 
-			message = fmt.Sprintf("<b>%s</b><br/>Traveling %s at<br/> %s mph as of %s", vehicle.VehicleName, CardinalDirection(&update.Heading), speed, lastUpdate)
+			// Query Routes that matches the update's RouteID
+
+			err = App.db.Routes.Find(bson.M{"id": update.Route}).Limit(1).One(&currentRoute)
```
Comment (JCBird1012) [inline api/vehicles.go]: Ask me why I'm not handling the error here! Apparently, ```err```will never **not** be ```nil```, and Go won't let you set HTTP headers twice. If you find a better way to handle a potential error here, let me know.

**Comment by jlyon1** [comment]
When a shuttle is not on a route this outputs "Traveling North-West on the route at 10.1 mph as of 9:34:05am."

**Comment by JCBird1012** [comment]
What's funny is that I even thought about that, but apparently forgot to handle it. 

I'll add a check to see if a shuttle's current route doesn't exist, I'll change the wording of the message so that it makes sense.

**Comment by kochman** [comment]
Yeah, and that might be why `api/vehicles.go`#L191 is not `nil`? I think we need to check for `mgo.ErrNotFound`.

**Comment by JCBird1012** [comment]
I was trying to handle the error as is done at line 168 right now. Apparently, the error headers are always set because ```err``` is never equal to ```nil```. So, I like your suggestion.

**Comment by kochman** [comment]
So as discussed, we're going to do this on the frontend and rip out the whole `/updates/message` endpoint.

**Comment by jlyon1** [comment]
Are we good to close this PR? I think its safer to say we just address the issues in a future PR where we handle message generation client side @JCBird1012 @kochman

**Comment by kochman** [comment]
Yeah, let's do that.

**Pull request closed without merging**